<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(197, 225, 165);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="table_nourritures_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>741</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(197, 225, 165);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>gestion animaux</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>741</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(197, 225, 165);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>ajouter</string>
       </attribute>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>70</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;type&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>100</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;sexe&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>190</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;prix&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>220</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 170, 0);
border-radius: 10px 100px / 120px;
color: rgb(255, 255, 255);
background-color: rgb(102, 152, 1);</string>
        </property>
        <property name="text">
         <string>ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_supp">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>250</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 10px 100px / 120px;
background-color: rgb(102, 152, 1);

color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>supprimer</string>
        </property>
       </widget>
       <widget class="QComboBox" name="le_sexe">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>100</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(225, 205, 183, 255), stop:1 rgba(255, 255, 255, 255));
selection-background-color: rgb(255, 170, 127);</string>
        </property>
        <item>
         <property name="text">
          <string>male</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>femelle</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>date de naissance</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="la_date">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>130</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(225, 205, 183, 255), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>id_animaux</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_id">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>40</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(225, 205, 183, 255), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>220</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 170, 0);
border-radius: 10px 25px ;

color: rgb(255, 255, 255);
background-color: rgb(102, 152, 1);</string>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
       <widget class="QTableView" name="table_animaux">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>40</y>
          <width>471</width>
          <height>201</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(225, 205, 183, 255), stop:1 rgba(255, 255, 255, 255));
selection-background-color: rgb(255, 170, 127);
border-top-color: rgb(85, 85, 255);
</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_prix">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>190</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(225, 205, 183, 255), stop:1 rgba(255, 255, 255, 255));
</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>10</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 170, 0);
border-radius: 10px 100px / 120px;

color: rgb(255, 255, 255);
background-color: rgb(102, 152, 1);
</string>
        </property>
        <property name="text">
         <string>tri croissant</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>10</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 170, 0);
background-color: rgb(102, 152, 1);
border-radius: 10px 100px / 120px;

color: rgb(255, 255, 255);
</string>
        </property>
        <property name="text">
         <string>tri decroissant</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>250</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 170, 0);
background-color: rgb(102, 152, 1);
border-radius: 10px 100px / 120px;

color: rgb(255, 255, 255);
</string>
        </property>
        <property name="text">
         <string>recherche</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>250</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(225, 205, 183, 255), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_9">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>160</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(225, 205, 183, 255), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
        <property name="text">
         <string>carnet de vaccination</string>
        </property>
       </widget>
       <widget class="QComboBox" name="le_type">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>70</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(225, 205, 183, 255), stop:1 rgba(255, 255, 255, 255));
selection-background-color: rgb(255, 170, 127);


</string>
        </property>
        <item>
         <property name="text">
          <string>vache</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mouton</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>poulet</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>oiseau</string>
         </property>
        </item>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="ajouter">
     <attribute name="title">
      <string>gestion nourritures</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>671</width>
        <height>371</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/Downloads/89034140_630747411034849_6413585486710833152_n (1).jpg);</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>671</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>621</width>
        <height>331</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_13">
       <attribute name="title">
        <string>ajouter</string>
       </attribute>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;id_nourriture&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_id_nourr">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;nom&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>47</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;quantité&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;prix_unitaire&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="la_qt">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>120</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="ajout_nourr">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>180</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>ajouter</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_prix_un">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>150</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableView" name="table_nourritures">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>40</y>
          <width>261</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="supp_nourr">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>270</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>180</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
       <widget class="QComboBox" name="le_nom">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>90</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>blé</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>orge</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mais</string>
         </property>
        </item>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
